core/src/main/java/nl/nn/adapterframework/http/IbisWebServiceSender.java

```diff
@@ -39,9 +40,8 @@
  * @author Gerrit van Brakel
  * @since 4.2
  */
-public class IbisWebServiceSender implements ISender, HasPhysicalDestination {
+public class IbisWebServiceSender extends SenderBase implements ISender, HasPhysicalDestination {
```

Comment:
When it extends SenderBase, it is not necessary to explicitly implement ISender, that comes with SenderBase